Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/core/impl/ConceptManager.java

```diff
@@ -477,10 +476,8 @@ public void deleteConcept(String id, String userName) throws LuceneException, In
     @Override
     public ConceptEntry getConceptWrappedEntryByWordNetId(String wordNetID)
             throws IllegalAccessException, LuceneException, IndexerRunningException {
-        Map<String, String> fieldMap = new HashMap<>();
-        fieldMap.put(LuceneFieldNames.WORDNETID, wordNetID);
-        ConceptEntry[] entries = indexService.searchForConcepts(fieldMap, SearchParamters.OP_AND);
-        for (ConceptEntry entry : entries) {
+        List<ConceptEntry> conceptEntries = client.getConceptByWordnetId(wordNetID);
+        for (ConceptEntry entry : conceptEntries) {
             // Wordnet is also added because lucene doesn't do an exact search
             // on fields
             if (entry.getId().contains("CON") && entry.getWordnetId().contains(wordNetID)) {
```

Comment:
Also, the id should start with "CON" and isn't there a method that returns "CON" from where the ids are created? This is not very good design. What if we change how the ids are created again?